backend/getData.js

```diff
@@ -112,6 +112,15 @@ function getData(element: Object): DataType {
     }
   }
 
+  if (typeof element.setNativeProps === 'function') {
```

Comment:
Note to self: this depends on how we expose host instances in the end.

It might be that we'll just expose this from RN renderer instead, and it will have to inject `setNativeProps(tag)` just like it does with `resolveStyle()`.